### PR TITLE
BDRE-169 Source code quality correction in md-commons

### PR DESCRIPTION
**Pull request by GithubMrinmoy**
@bdrekapil @mishisinku Please review.

### CODE REVIEW

File: md-commons/src/main/java/com/wipro/ats/bdre/BaseStructure.java [hunk 1]
```diff
@@ -93,6 +89,7 @@ public final CommandLine getCommandLine(String[] params, String[][] paramStructu
             return cmd;
 
         } catch (ParseException e) {
+            LOGGER.error(e.getMessage());
             //System.err cannot be removed as it is an command line argument
             printUsage("java <main_class> ", e.getMessage(), options, System.err);
```
Comment (bdre-bda) [inline md-commons/src/main/java/com/wipro/ats/bdre/BaseStructure.java]: MAJOR Replace this usage of System.out or System.err by a logger. [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS106)

File: md-commons/src/main/java/com/wipro/ats/bdre/BaseStructure.java [hunk 2]
```diff
@@ -93,6 +89,7 @@ public final CommandLine getCommandLine(String[] params, String[][] paramStructu
...
             System.exit(1);
```
Comment (bdre-bda) [inline md-commons/src/main/java/com/wipro/ats/bdre/BaseStructure.java]: CRITICAL Remove this call to "exit" or ensure it is really required. [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1147)

**Comment by bdre-bda** [comment]
SonarQube analysis reported 15 issues:
* BLOCKER 2 blocker
* CRITICAL 2 critical
* MAJOR 2 major
* MINOR 9 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [The Cyclomatic Complexity of this method "populateBeans" is 16 which is greater than 10 authorized.](https://github.com/WiproOpenSourcePractice/openbdre/blob/df52e120967e8c6a0851c3985076e4ec9afe9918/data-lineage/src/main/java/com/wipro/ats/bdre/lineage/api/PersistenceUnit.java#L85) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)
* MINOR [Replace all tab characters in this file by sequences of white-spaces.](https://github.com/WiproOpenSourcePractice/openbdre/blob/df52e120967e8c6a0851c3985076e4ec9afe9918/data-lineage/src/main/java/com/wipro/ats/bdre/lineage/api/PersistenceUnit.java) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00105)
* CRITICAL [Remove this hard-coded password.](https://github.com/WiproOpenSourcePractice/openbdre/blob/df52e120967e8c6a0851c3985076e4ec9afe9918/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/DQSetupInfo.java#L33) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS2068)
* MINOR [Move this constructor to comply with Java Code Conventions.](https://github.com/WiproOpenSourcePractice/openbdre/blob/df52e120967e8c6a0851c3985076e4ec9afe9918/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/ProcessInfo.java#L117) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/WiproOpenSourcePractice/openbdre/blob/df52e120967e8c6a0851c3985076e4ec9afe9918/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/ProcessInfo.java#L96) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this constructor to comply with Java Code Conventions.](https://github.com/WiproOpenSourcePractice/openbdre/blob/df52e120967e8c6a0851c3985076e4ec9afe9918/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/ProcessInfo.java#L101) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)
* MAJOR [Method has 9 parameters, which is greater than 7 authorized.](https://github.com/WiproOpenSourcePractice/openbdre/blob/df52e120967e8c6a0851c3985076e4ec9afe9918/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/ProcessInfo.java#L117) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00107)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/WiproOpenSourcePractice/openbdre/blob/df52e120967e8c6a0851c3985076e4ec9afe9918/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/table/ArchiveConsumpQueue.java#L117) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/WiproOpenSourcePractice/openbdre/blob/df52e120967e8c6a0851c3985076e4ec9afe9918/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/table/BatchStatus.java#L69) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)
* BLOCKER ["@" is not available at runtime and cannot be seen with reflection.](https://github.com/WiproOpenSourcePractice/openbdre/blob/df52e120967e8c6a0851c3985076e4ec9afe9918/md-commons/src/main/java/com/wipro/ats/bdre/util/OozieUtil.java#L65) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS2109)
* ... 3 more

**Comment by bdre-bda** [comment]
SonarQube analysis reported 13 issues:
* CRITICAL 2 critical
* MAJOR 2 major
* MINOR 9 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Replace all tab characters in this file by sequences of white-spaces.](https://github.com/WiproOpenSourcePractice/openbdre/blob/df52e120967e8c6a0851c3985076e4ec9afe9918/data-lineage/src/main/java/com/wipro/ats/bdre/lineage/api/PersistenceUnit.java) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00105)
* MINOR [The Cyclomatic Complexity of this method "populateBeans" is 16 which is greater than 10 authorized.](https://github.com/WiproOpenSourcePractice/openbdre/blob/df52e120967e8c6a0851c3985076e4ec9afe9918/data-lineage/src/main/java/com/wipro/ats/bdre/lineage/api/PersistenceUnit.java#L85) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)
* CRITICAL [Remove this hard-coded password.](https://github.com/WiproOpenSourcePractice/openbdre/blob/df52e120967e8c6a0851c3985076e4ec9afe9918/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/DQSetupInfo.java#L33) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS2068)
* MAJOR [Method has 9 parameters, which is greater than 7 authorized.](https://github.com/WiproOpenSourcePractice/openbdre/blob/df52e120967e8c6a0851c3985076e4ec9afe9918/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/ProcessInfo.java#L117) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00107)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/WiproOpenSourcePractice/openbdre/blob/df52e120967e8c6a0851c3985076e4ec9afe9918/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/ProcessInfo.java#L96) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this constructor to comply with Java Code Conventions.](https://github.com/WiproOpenSourcePractice/openbdre/blob/df52e120967e8c6a0851c3985076e4ec9afe9918/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/ProcessInfo.java#L101) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this constructor to comply with Java Code Conventions.](https://github.com/WiproOpenSourcePractice/openbdre/blob/df52e120967e8c6a0851c3985076e4ec9afe9918/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/ProcessInfo.java#L117) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/WiproOpenSourcePractice/openbdre/blob/df52e120967e8c6a0851c3985076e4ec9afe9918/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/table/ArchiveConsumpQueue.java#L117) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/WiproOpenSourcePractice/openbdre/blob/df52e120967e8c6a0851c3985076e4ec9afe9918/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/table/BatchStatus.java#L69) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Refactor this code to not nest more than 3 if/for/while/switch/try statements.](https://github.com/WiproOpenSourcePractice/openbdre/blob/df52e120967e8c6a0851c3985076e4ec9afe9918/md-commons/src/main/java/com/wipro/ats/bdre/util/OozieUtil.java#L90) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS134)
* ... 1 more

**Comment by bdre-bda** [comment]
SonarQube analysis reported 13 issues:
* CRITICAL 2 critical
* MAJOR 2 major
* MINOR 9 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [The Cyclomatic Complexity of this method "populateBeans" is 16 which is greater than 10 authorized.](https://github.com/WiproOpenSourcePractice/openbdre/blob/d39253e822b85f0353acd0190d44064fc92f4a0b/data-lineage/src/main/java/com/wipro/ats/bdre/lineage/api/PersistenceUnit.java#L85) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)
* MINOR [Replace all tab characters in this file by sequences of white-spaces.](https://github.com/WiproOpenSourcePractice/openbdre/blob/d39253e822b85f0353acd0190d44064fc92f4a0b/data-lineage/src/main/java/com/wipro/ats/bdre/lineage/api/PersistenceUnit.java) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00105)
* CRITICAL [Remove this hard-coded password.](https://github.com/WiproOpenSourcePractice/openbdre/blob/d39253e822b85f0353acd0190d44064fc92f4a0b/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/DQSetupInfo.java#L33) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS2068)
* MAJOR [Method has 9 parameters, which is greater than 7 authorized.](https://github.com/WiproOpenSourcePractice/openbdre/blob/d39253e822b85f0353acd0190d44064fc92f4a0b/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/ProcessInfo.java#L117) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00107)
* MINOR [Move this constructor to comply with Java Code Conventions.](https://github.com/WiproOpenSourcePractice/openbdre/blob/d39253e822b85f0353acd0190d44064fc92f4a0b/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/ProcessInfo.java#L101) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/WiproOpenSourcePractice/openbdre/blob/d39253e822b85f0353acd0190d44064fc92f4a0b/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/ProcessInfo.java#L96) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this constructor to comply with Java Code Conventions.](https://github.com/WiproOpenSourcePractice/openbdre/blob/d39253e822b85f0353acd0190d44064fc92f4a0b/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/ProcessInfo.java#L117) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/WiproOpenSourcePractice/openbdre/blob/d39253e822b85f0353acd0190d44064fc92f4a0b/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/table/ArchiveConsumpQueue.java#L117) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/WiproOpenSourcePractice/openbdre/blob/d39253e822b85f0353acd0190d44064fc92f4a0b/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/table/BatchStatus.java#L69) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [The Cyclomatic Complexity of this method "persistSingleBean" is 22 which is greater than 10 authorized.](https://github.com/WiproOpenSourcePractice/openbdre/blob/d39253e822b85f0353acd0190d44064fc92f4a0b/md-commons/src/main/java/com/wipro/ats/bdre/util/OozieUtil.java#L61) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)
* ... 1 more

**Comment by bdre-bda** [comment]
SonarQube analysis reported 12 issues:
* CRITICAL 1 critical
* MAJOR 2 major
* MINOR 9 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [The Cyclomatic Complexity of this method "populateBeans" is 16 which is greater than 10 authorized.](https://github.com/WiproOpenSourcePractice/openbdre/blob/82fff05e82bcff61374765006c3a9af4952a8558/data-lineage/src/main/java/com/wipro/ats/bdre/lineage/api/PersistenceUnit.java#L85) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)
* MINOR [Replace all tab characters in this file by sequences of white-spaces.](https://github.com/WiproOpenSourcePractice/openbdre/blob/82fff05e82bcff61374765006c3a9af4952a8558/data-lineage/src/main/java/com/wipro/ats/bdre/lineage/api/PersistenceUnit.java) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00105)
* CRITICAL [Remove this hard-coded password.](https://github.com/WiproOpenSourcePractice/openbdre/blob/82fff05e82bcff61374765006c3a9af4952a8558/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/DQSetupInfo.java#L33) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS2068)
* MINOR [Move this constructor to comply with Java Code Conventions.](https://github.com/WiproOpenSourcePractice/openbdre/blob/82fff05e82bcff61374765006c3a9af4952a8558/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/ProcessInfo.java#L117) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)
* MAJOR [Method has 9 parameters, which is greater than 7 authorized.](https://github.com/WiproOpenSourcePractice/openbdre/blob/82fff05e82bcff61374765006c3a9af4952a8558/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/ProcessInfo.java#L117) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00107)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/WiproOpenSourcePractice/openbdre/blob/82fff05e82bcff61374765006c3a9af4952a8558/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/ProcessInfo.java#L96) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this constructor to comply with Java Code Conventions.](https://github.com/WiproOpenSourcePractice/openbdre/blob/82fff05e82bcff61374765006c3a9af4952a8558/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/ProcessInfo.java#L101) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/WiproOpenSourcePractice/openbdre/blob/82fff05e82bcff61374765006c3a9af4952a8558/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/table/ArchiveConsumpQueue.java#L117) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/WiproOpenSourcePractice/openbdre/blob/82fff05e82bcff61374765006c3a9af4952a8558/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/table/BatchStatus.java#L69) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [The Cyclomatic Complexity of this method "persistSingleBean" is 22 which is greater than 10 authorized.](https://github.com/WiproOpenSourcePractice/openbdre/blob/82fff05e82bcff61374765006c3a9af4952a8558/md-commons/src/main/java/com/wipro/ats/bdre/util/OozieUtil.java#L61) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)
* ... 1 more

File: md-commons/src/main/java/com/wipro/ats/bdre/BaseStructure.java (hunk 1 above)
Comment (wiprodev) [inline md-commons/src/main/java/com/wipro/ats/bdre/BaseStructure.java:93]: should be LOGGER.error(e)

File: md-commons/src/main/java/com/wipro/ats/bdre/BaseStructure.java [hunk 3]
```diff
@@ -90,14 +86,13 @@ public final CommandLine getCommandLine(String[] params, String[][] paramStructu
             }
 
             cmd = parser.parse(options, newParams.toArray(new String[]{}));
-            return cmd;
 
         } catch (ParseException e) {
+            LOGGER.error(e);
             //System.err cannot be removed as it is an command line argument
             printUsage("java <main_class> ", e.getMessage(), options, System.err);
```
Comment (bdre-bda) [inline md-commons/src/main/java/com/wipro/ats/bdre/BaseStructure.java:93]: MAJOR Replace this usage of System.out or System.err by a logger. [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS106)

**Comment by bdre-bda** [comment]
SonarQube analysis reported 11 issues:
* MAJOR 2 major
* MINOR 9 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Replace all tab characters in this file by sequences of white-spaces.](https://github.com/WiproOpenSourcePractice/openbdre/blob/a9c21dac0ef80964599111f158ec44bf1452f47b/data-lineage/src/main/java/com/wipro/ats/bdre/lineage/api/PersistenceUnit.java) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00105)
* MINOR [The Cyclomatic Complexity of this method "populateBeans" is 16 which is greater than 10 authorized.](https://github.com/WiproOpenSourcePractice/openbdre/blob/a9c21dac0ef80964599111f158ec44bf1452f47b/data-lineage/src/main/java/com/wipro/ats/bdre/lineage/api/PersistenceUnit.java#L85) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)
* MINOR [Move this constructor to comply with Java Code Conventions.](https://github.com/WiproOpenSourcePractice/openbdre/blob/a9c21dac0ef80964599111f158ec44bf1452f47b/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/ProcessInfo.java#L101) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this constructor to comply with Java Code Conventions.](https://github.com/WiproOpenSourcePractice/openbdre/blob/a9c21dac0ef80964599111f158ec44bf1452f47b/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/ProcessInfo.java#L117) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/WiproOpenSourcePractice/openbdre/blob/a9c21dac0ef80964599111f158ec44bf1452f47b/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/ProcessInfo.java#L96) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)
* MAJOR [Method has 9 parameters, which is greater than 7 authorized.](https://github.com/WiproOpenSourcePractice/openbdre/blob/a9c21dac0ef80964599111f158ec44bf1452f47b/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/ProcessInfo.java#L117) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00107)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/WiproOpenSourcePractice/openbdre/blob/a9c21dac0ef80964599111f158ec44bf1452f47b/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/table/ArchiveConsumpQueue.java#L117) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/WiproOpenSourcePractice/openbdre/blob/a9c21dac0ef80964599111f158ec44bf1452f47b/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/table/BatchStatus.java#L69) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Refactor this code to not nest more than 3 if/for/while/switch/try statements.](https://github.com/WiproOpenSourcePractice/openbdre/blob/a9c21dac0ef80964599111f158ec44bf1452f47b/md-commons/src/main/java/com/wipro/ats/bdre/util/OozieUtil.java#L90) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS134)
* MINOR [The Cyclomatic Complexity of this method "persistSingleBean" is 22 which is greater than 10 authorized.](https://github.com/WiproOpenSourcePractice/openbdre/blob/a9c21dac0ef80964599111f158ec44bf1452f47b/md-commons/src/main/java/com/wipro/ats/bdre/util/OozieUtil.java#L61) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)
* ... 0 more

**Comment by mishisinku** [comment]
@GithubMrinmoy  please update with predevelop again

**Comment by bdre-bda** [comment]
SonarQube analysis reported 11 issues:
- MAJOR 2 major
- MINOR 9 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
- MINOR [The Cyclomatic Complexity of this method "populateBeans" is 16 which is greater than 10 authorized.](https://github.com/WiproOpenSourcePractice/openbdre/blob/c5b52c61a176f19e582290908a2b100d4de4d2dc/data-lineage/src/main/java/com/wipro/ats/bdre/lineage/api/PersistenceUnit.java#L85) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)
- MINOR [Replace all tab characters in this file by sequences of white-spaces.](https://github.com/WiproOpenSourcePractice/openbdre/blob/c5b52c61a176f19e582290908a2b100d4de4d2dc/data-lineage/src/main/java/com/wipro/ats/bdre/lineage/api/PersistenceUnit.java) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00105)
- MINOR [Move this constructor to comply with Java Code Conventions.](https://github.com/WiproOpenSourcePractice/openbdre/blob/c5b52c61a176f19e582290908a2b100d4de4d2dc/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/ProcessInfo.java#L101) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)
- MINOR [Move this constructor to comply with Java Code Conventions.](https://github.com/WiproOpenSourcePractice/openbdre/blob/c5b52c61a176f19e582290908a2b100d4de4d2dc/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/ProcessInfo.java#L117) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)
- MAJOR [Method has 9 parameters, which is greater than 7 authorized.](https://github.com/WiproOpenSourcePractice/openbdre/blob/c5b52c61a176f19e582290908a2b100d4de4d2dc/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/ProcessInfo.java#L117) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00107)
- MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/WiproOpenSourcePractice/openbdre/blob/c5b52c61a176f19e582290908a2b100d4de4d2dc/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/ProcessInfo.java#L96) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)
- MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/WiproOpenSourcePractice/openbdre/blob/c5b52c61a176f19e582290908a2b100d4de4d2dc/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/table/ArchiveConsumpQueue.java#L117) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)
- MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/WiproOpenSourcePractice/openbdre/blob/c5b52c61a176f19e582290908a2b100d4de4d2dc/md-commons/src/main/java/com/wipro/ats/bdre/md/beans/table/BatchStatus.java#L69) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)
- MINOR [Refactor this code to not nest more than 3 if/for/while/switch/try statements.](https://github.com/WiproOpenSourcePractice/openbdre/blob/c5b52c61a176f19e582290908a2b100d4de4d2dc/md-commons/src/main/java/com/wipro/ats/bdre/util/OozieUtil.java#L90) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS134)
- MINOR [The Cyclomatic Complexity of this method "persistSingleBean" is 22 which is greater than 10 authorized.](https://github.com/WiproOpenSourcePractice/openbdre/blob/c5b52c61a176f19e582290908a2b100d4de4d2dc/md-commons/src/main/java/com/wipro/ats/bdre/util/OozieUtil.java#L61) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)
- ... 0 more

**Pull request merged**